wielder/util/hocon_util.py

```diff
@@ -121,3 +121,18 @@ def conf_to_native(conf, vessel={}):
 
     return vessel
 
+
+def merge_configurations(conf, remote_conf):
```

Comment:
The name indicates a broad use, It should explain why you found it necessary and useful to abstract. Is this really repeatable and for which special cases?